### ISSUE TITLE
distributed trust

### ISSUE THREAD

**Issue by adrelanos** [ioerror/tlsdate#112]
From the [Tails Time Syncing Design](https://tails.boum.org/contribute/design/Time_syncing/#index4h2) page.

_HTP source pools_

_What sources should be trusted? [...]_

_The HTP pools used by Tails are based on stable and reliable webservers that get great amounts of traffic. They are categorized into three different pools according to their members relationship to the members in the other pools; any member in a one pool should be unlikely to share logs (or other identifying data), or to agree to send fake time information, with a member from the the other pools. The pools are as follows:_
- _The "pal" pool are run by groups that are likely to take great care of their visitors' privacy._
- _The "foe" pool are managed by adversaries of the "pal" pool._
- _The "neutral" pool members have a neutral relationship to both the "pal" and "foe" pool._

_The pools are listed in [config/chroot local-includes/etc/default/htpdate](http://git.immerda.ch/?p=amnesia.git;a=blob_plain;f=config%2Fchroot%5Flocal%2Dincludes%2Fetc%2Fdefault%2Fhtpdate)._

_Basically, Tails htpdate pick three random servers (one from each pool), and then build the mediate of the three advertised dates._

Could you please add such a feature to tlsdate as well?
